packages/core/src/components/DataTable/index.tsx

```diff
@@ -370,19 +418,26 @@ export class DataTable extends React.Component<DataTableProps & WithStylesProps,
   rowGetter = (expandedDataList: ExpandedRow[]) => ({ index }: { index: number }) =>
     expandedDataList[index];
 
+  cache = new CellMeasurerCache({
+    fixedHeight: false,
+    fixedWidth: true,
+    defaultHeight: 16,
```

Comment:
In thinking about how this will all be used in practice, I think `defaultHeight` and `minHeight` are going to be needed in order to have more pixel-perfect control over the dynamic heights.

can you add these as props with defaults? something like `defaultDynamicRowHeight: 16, minimumDynamicRowHeight: undefined`. I can add this in a follow up PR but I think it'd be nice to save a review cycle.

Reply:
Added